src/ddmd/backend/dwarf.d

```diff
@@ -1,4 +1,4 @@
-/* Dwarf debug
+/** Dwarf debug
  *
  * Source: $(DMDSRC backend/_dwarf.d)
  */
```

Comment:
I know that it's a debug file, but I wasn't sure whether it was set "private" on purpose.
It's currently visible on the Ddoc build:

https://dlang.org/phobos/ddmd_backend_dwarf.html

But I guess we should simply add it to the [blacklist of hidden modules](https://github.com/dlang/dlang.org/blob/master/posix.mak#L73).